03LessonThree/homework/git-scm.html

```diff
@@ -7,21 +7,21 @@
 </head>
 
 <body>
-    <div id="header">
-        <div id="nav">
+    <header>
+        <nav>
             <span><a href="#">Home</a></span>
             <span><a href="#">How does it work</a></span>
             <span><a href="#">Contact Us</a></span>
-        </div>
-    </div>
-    <div id="main">
-        <div id="section">
+        </nav>
+    </header>
+    <main>
+        <section>
             <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
             <h2>Welcome to Git</h2>
             <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
-        </div>
-        <div id="section">
-            <div id="article">
+        </section>
+        <section>
+            <article>
                 <h3>About</h3>

```

Comment:
Great! Indenting you code like you have done here makes it easier for your future self and other developers looking at your code. Makes it readable and easy follow where the start and close tags are for the elements.